flutter_local_notifications/pubspec.yaml

```diff
@@ -9,8 +9,10 @@ dependencies:
   clock: ^1.1.0
   flutter:
     sdk: flutter
-  flutter_local_notifications_linux: ^0.4.1
-  flutter_local_notifications_platform_interface: ^6.0.0
+  flutter_local_notifications_linux: #^0.4.1
```

Comment:
Could you please revert all the pubspec.yaml changes? It's not necessary I suspect the reason you did these changes was so you could test more easily. If so, note this plugin recently changed to use [melos](https://melos.invertase.dev). If you bootstrap melos then it will do changes behind the scenes to use the local dependencies